site/en/blog/crx-scripting-api/index.md

```diff
@@ -0,0 +1,209 @@
+---
+title: "Introducing the chrome.scripting API"
+# description: >
+#   Why and how we implemented color vision deficiency simulation in DevTools and the Blink Renderer.
+layout: "layouts/blog-post.njk"
+authors:
+  - dotproto
+date: 2021-05-03
+hero: 'image/dPDCek3EhZgLQPGtEG3y0fTn4v82/HD1U7BxDVRDcTxNaQJ4C.jpg'
+alt: ''
+tags:
+  - extensions
+---
+
+Manifest V3 introduces a number of changes to Chrome's extension platform, a handful of which are
+notable or significant changes. In this post, we'll be exploring the motivations and changes
+introduced by one of the more notable changes: the introduction of the `chrome.scripting` API.
+
+## What is `chrome.scripting`?
+
+As the name might suggest, `chrome.scripting` is a new namespace introduced in Manifest V3
+responsible for script and style injection capabilities.
+
+Developers that have created Chrome extensions in the past may be familiar with Manifest V2 methods
+on the Tabs API like `chrome.tabs.executeScript` and `chrome.tabs.insertCSS`. These methods allow
+extensions to inject scripts and stylesheets into pages, respectively. In Manifest V3, these
+capabilities have moved to `chrome.scripting` and we plan to expand this API with some new
+capabilities in the future.
+
+## Why create a new API?
+
+With a change like this, one of the first questions that tends to come up is, "why?"
+
+A few different factors lead to the Chrome team deciding to introduce a new namespace for scripting.
+First, the Tabs API is a bit of a junk drawer for features. Second, we needed to make breaking
+changes to the existing `executeScript` API. Third, we knew that we wanted to expand scripting
+capabilities for extensions. Together, these concerns clearly defined a need for a new namespace to
+house scripting capabilities.
+
+### The junk drawer
+
+One of the issues that has been bothering the Extensions Team for the past few years is that the
+`chrome.tabs` API is overloaded. When this API was first introduced most of the capabilities it
+provided were related to the broad concept of a browser tab. Even at that point, though, it was a
+bit of a grab bag of features and over the years this collection has only grown.
+
+[Image: a beautiful junk drawer. Caption: Artist's rendition of the Tabs API.]
+
+By the time Manifest V3 was released, the Tabs API had grown to cover basic tab management,
+selection management, window organization, positioning, messaging, zoom control, basic navigation,
```

Comment:
```suggestion
selection management, window organization, messaging, zoom control, basic navigation,
```

(I don't see any position in the tabs API - I think that's just windows)

Reply:
You're right, my mistake.